docs/SWTPM.md

```diff
@@ -0,0 +1,40 @@
+# Using wolfTPM with SWTPM
+
+wolfTPM is to be able to interface with SW TPM interfaces defined by section D.3 of
+[TPM-Rev-2.0-Part-4-Supporting-Routines-01.38-code](https://trustedcomputinggroup.org/wp-content/uploads/TPM-Rev-2.0-Part-4-Supporting-Routines-01.38-code.pdf)
+
+Using the socket connection for SWTPM is exclusive and not compatible with TIS or devtpm.
+
+Only a subset of functionality is implemented to support testing of wolfTPM. The platform requests are not used by wolfTPM.
+
+Two implementations were used in testing:
+* http://ibmswtpm.sourceforge.net/
+* https://github.com/stefanberger/swtpm
```

Comment:
Would be great to add build instructions for this one as well.

Reply:
Done, harder than it should have been to reproduce.